### PR TITLE
#68 Docker Container

### PR DESCRIPTION
**Pull request by gibboj**
Added docker for easier development setup

### CODE REVIEW

**Comment by vnugent** [comment]
Thanks so much for the PR!   
I didn't mention anywhere in the README that `develop` is the default branch where we start our work from.  My apologies.  Would you be able to resubmit a new PR starting from `develop` branch?

You can cherry pick commit `88ca5f3` to the new branch.  For example:
```
git checkout develop
git checkout -b docker-container2
git cherry-pick 88ca5f3
```

**Comment by gibboj** [comment]
@vnugent I rebased on develop. Let me know what you think

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: "3"
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./src:/myapp/src # Links the source files to the running container
```
Comment (vnugent) [inline docker-compose.yml:8]: I'd also map `node_modules` to the host to keep our docker image small
```
    volumes:
      - ./src:/myapp/src # Links the source files to the running container
      - ./node_modules:/myapp/node_modules
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+FROM node:16-bullseye-slim as builder
+
+RUN apt-get update -y
+RUN apt-get install -y gcc make musl-dev libvips-dev libvips
+RUN yarn global add gatsby-cli
```
Comment (vnugent) [inline Dockerfile:6]: we can access `gatsby-cli` which is already installed in `node_modules/.bin`

```

FROM node:16-bullseye-slim as builder

ENV GATSBY_TELEMETRY_DISABLED=1
ENV PATH /myapp/node_modules/.bin:$PATH

RUN apt-get update -y
# The port gatsby runs on
EXPOSE 8000
WORKDIR /myapp
COPY ./package.json /myapp
COPY ./yarn.lock /myapp
COPY . /myapp
CMD ["sh", "-c", "yarn install && gatsby develop -H 0.0.0.0"]

```

**Comment by vnugent** [comment]
@allcontributors please add @gibboj as a contributor for code

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.cache/
+node_modules/
+public/
```
Comment (vnugent) [inline .dockerignore:3]: can you please add `opentacos-content`

**Review by vnugent — CHANGES_REQUESTED** [review]
@gibboj can you please make the 3 changes and push again? then we're ready to merge. Thanks!

**Pull request merged**